src/index.html

```diff
@@ -1,12 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__month calendar__month--month-length--28">
+
+        <div class="calendar__start-day calendar__start-day--thu"></div>
+
+        <div class="calendar-cell calendar-cell--1"></div>
```

Comment:
```suggestion
        <div class="calendar-cell"></div>
```
There is no need in an additional class here